frontend/components/sections/BannersSection/MultipleBanners.tsx

```diff
@@ -0,0 +1,98 @@
+import {
+   AspectRatio,
+   Box,
+   Flex,
+   Heading,
+   Link,
+   SimpleGrid,
+} from '@chakra-ui/react';
+import { faArrowRight } from '@fortawesome/pro-solid-svg-icons';
+import { FaIcon } from '@ifixit/icons';
+import { ResponsiveImage } from '@ifixit/ui';
+import type { Banner } from '@models/components/banner';
+
+export interface MultipleBannersProps {
+   banners: Banner[];
+}
+
+export function MultipleBanners({ banners }: MultipleBannersProps) {
+   return (
+      <Box as="section" position="relative" w="full">
+         <SimpleGrid
+            columns={{
+               base: 1,
+               md: banners.length,
+            }}
+            pb="10"
+         >
+            {banners.map((banner) => {
+               return <BannerGridItem key={banner.id} banner={banner} />;
+            })}
+         </SimpleGrid>
+      </Box>
+   );
+}
+
+interface BannerProps {
+   banner: Banner;
+}
+
+function BannerGridItem({ banner }: BannerProps) {
+   return (
+      <Flex direction="column">
+         {banner.image && (
+            <AspectRatio ratio={4 / 3}>
+               <ResponsiveImage
+                  src={banner.image.url}
+                  alt="store hero image"
```

Comment:
Same here

```suggestion
                  alt={banner.image.alternativeText ?? "store hero image"}
```

Reply:
Here I forgot to put `""` (part of this code is coming from another branch). This is kind of the same concept as before, although It's not a background image here, so one could argue that here it's reasonable to expect the screen reader to see the image.

Reply:
I would still consider these images decorative, and they have associated descriptive text, so I agree that this could also have `alt=""` like the single banner.